docker/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.1'
+
+services:
+  identities_db:
+    image: postgres:11-alpine
+    volumes:
+      - $HOME/docker/volumes/postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: identities
+
+  identities_db_client:
+    image:  dpage/pgadmin4:latest
+    environment:
+      PGADMIN_DEFAULT_EMAIL: admin@labengine.com
+      PGADMIN_DEFAULT_PASSWORD: admin
+    ports:
+      - 80:80
+    depends_on:
+      - identities_db
+
+  oauth:
+    build:
+      context: ../
```

Comment:
are you sure that you need to specify one level up when you are running `docker-compose` from the root?